Makefile

```diff
@@ -16,6 +16,7 @@ BUILD_CONTAINER_NAME=calico/cni_build_container
 BUILD_CONTAINER_MARKER=cni_build_container.created
 DEPLOY_CONTAINER_NAME=calico/cni
 DEPLOY_CONTAINER_MARKER=cni_deploy_container.created
+TEMP_DIR:=$(shell mktemp -d)
```

Comment:
yes, but I figured being a temp dir it would not make much impact.  I'm open to just putting into a local dir and adding it to the make clean if that is preferable?